src/main/java/tools/map/xml/creator/MapXmlHelper.java

```diff
@@ -1486,4 +1381,19 @@ public static String getHtmlStringFromCanalDefinitions() {
     return sb.toString();
   }
 
+  static class Wrapper<T> {
```

Comment:
Okay, enough.

Yeah, it is a balance. Generic code is more costly though (roughly 3x as a rule of thumb per some youtube videos I watched some time ago, YMMV), so generic without a reason is not good. Then again, we do use `Map` instead of `HashMap`.